code/game/gamemodes/objective.dm

```diff
@@ -225,7 +225,7 @@
 		if(player.mind && player.mind != owner)
 			if(player.stat != DEAD)
 				switch(player.type)
-					if(/mob/living/silicon/ai, /mob/living/silicon/pai)
+					if(/mob/living/silicon) //Borgs are technically dead anyways

```

Comment:
Nope. Just replace the whole switch with  ``istype(player, /mob/living/silicon)``